### ISSUE TITLE
Cannot run bp4dbg

### ISSUE THREAD

**Issue by pnorbert** [ornladios/ADIOS2#2040]
**Describe the bug**
After installing adios2, I cannot run the bp4dbg utility because it fails with this error:
```
$ /opt/adios2/bin/bp4dbg localArray.bp/md.0
========================================================
    Metadata File: localArray.bp/md.0
========================================================
Traceback (most recent call last):
  File "/opt/adios2/bin/bp4dbg", line 105, in <module>
    DumpMetadataFiles(args)
  File "/opt/adios2/bin/bp4dbg", line 81, in DumpMetadataFiles
    DumpMetaData(fname)
  File "/opt/adios2/lib/python3/dist-packages/adios2/bp4dbg/metadata.py", line 607, in DumpMetaData
    status = bp4dbg_utils.ReadHeader(f, fileSize, "Metadata")
NameError: name 'bp4dbg_utils' is not defined
```

**Additional context**
The bp4dbg scripts use bp4dbg_utils._FunctionName()_ to call the utility functions but they import them as 

`from .utils import *`

The function calls cannot just be simply renamed to _FunctionName()_ because there will be clashes with other functions:

```
Traceback (most recent call last):
  File "/opt/adios2/bin/bp4dbg", line 105, in <module>
    DumpMetadataFiles(args)
  File "/opt/adios2/bin/bp4dbg", line 81, in DumpMetadataFiles
    DumpMetaData(fname)
  File "/opt/adios2/lib/python3/dist-packages/adios2/bp4dbg/metadata.py", line 607, in DumpMetaData
    status = ReadHeader(f, fileSize, "Metadata")
TypeError: ReadHeader() takes 1 positional argument but 3 were given
```

**Comment by pnorbert** [comment]
@chuckatkins What is the proper pythonesque way of fixing this?

**Comment by pnorbert** [comment]
This is fixed by PR #2073
